frontend/src/global/constants.ts

```diff
@@ -0,0 +1,31 @@
+export const OCIM_API_BASE = process.env.REACT_APP_OCIM_API_BASE || 'http://localhost:5000';
```

Comment:
Since these are static files, the environment variables are supposed to be read in during asset compilation. The compilation pipeline should prove the correct value for this during compilation.